app/models/main.py

```diff
@@ -1687,6 +1714,16 @@ def serialize(self):
 
         return purge_nulls_from_data(data)
 
+    def _parent_brief_fields(self, parent_brief):
+        return {
+            **self._unpack_fields(self.PARENT_BRIEF_FIELDS, parent_brief),
+            **{'framework': self._unpack_fields(self.FRAMEWORK_FIELDS, parent_brief)}
```

Comment:
Can we go to the underlying model instances rather than go via the existing serialization.

I don't want "frameworkFramework" nonsense to leak into our new more-perfect thing: we need to have a place to rename it to 'family'.

Reply:
Not sure I follow about the underlying model instances?

Kept `frameworkFramework` for consistency's sake however if we have decided to name it `family` it should start somewhere

Reply:
Hmm okay I've thought some more and we can fix several things at the same time here, I _think_:
 - go to the Brief serialization and add the `framework` key (i.e. do the duplicating thing, I genuinely don't think the slight increase in data volume is significant - we'll remove the old keys later)
 - here, then you can just add `framework` to `parent_brief_fields` and boom the data you need is here too?

Reply:
I like that idea @galund - hopefully it won't make too many more tests fail 😸 I don't think any of the consumers of serialized Briefs should care about an extra key.

Reply:
Now done. I included all the `Brief.frameworkFoo` fields in the `framework` sub dict so we can be consistent and use it for everything.